aspnetcore/signalr/hubs.md

```diff
@@ -120,6 +120,56 @@ By default, a server hub method name is the name of the .NET method. To change t
 
 :::code language="csharp" source="~/../AspNetCore.Docs.Samples/signalr/hubs/samples/6.x/SignalRHubsSample/Snippets/Hubs/ChatHub.cs" id="snippet_HubMethodName" highlight="1":::
 
+## Inject services into your hub
```

Comment:
To avoid 2nd person where possible ...

```suggestion
## Inject services into a hub
```

or it could be "Hub service injection" ... or similar.